Add nfl-team-schedule-widget (#16)
dersvenhesse/awesome-scriptable: README.md

- [fitx-widget.js](https://gist.github.com/DanielStefanK/487175b6f65ede401e37ee4848970176) - Workload of a FitX gym.

> - [nfl-team-schedule-widget](https://github.com/brianwalborn/nfl-team-schedule-widget) - the current season schedule for your favorite NFL team

Reviewer: ```suggestion
- [nfl-team-schedule-widget](https://github.com/brianwalborn/nfl-team-schedule-widget) - The current season schedule an NFL team.
```

Reviewer: Sorry for the delayed feedback. This suggestion should keep the linter happy.

Author: No worries! Addressed the missing punctuation.
